### PR TITLE
Add binary txt option.

### PR DESCRIPTION
**Pull request by kkushimoto**

### CODE REVIEW

**Comment by watson** [comment]
@kkushimoto Thanks for PR :smiley: 

Just a quick question: I'm really curious of your use case with binary TXT records. Do you mind sharing it?

File: lib/browser.js [hunk 1]
```diff
@@ -35,14 +35,21 @@ function Browser (mdns, opts, onup) {
...
 
   if (!opts || !opts.type) {
     this._name = WILDCARD
     this._wildcard = true
+    this._txt = dnsTxt()
   } else {
     this._name = serviceName.stringify(opts.type, opts.protocol || 'tcp') + TLD
     if (opts.name) this._name = opts.name + '.' + this._name
     this._wildcard = false
+    if (!opts.txt || !opts.txt.binary) {
```
Comment (watson) [inline lib/browser.js]: In case the `txt` property is given, why not just pass it into the `dnsTxt` constructor as it is?

**Comment by kkushimoto** [comment]
Thank you for reviewing my PR.

>  I'm really curious of your use case with binary TXT records.

I want to find a service that publish binary value txt.

 I'm not good at English :sweat:

File: lib/browser.js (hunk 1 above)
Comment (kkushimoto) [inline lib/browser.js]: Precisely!

**Comment by watson** [comment]
@kkushimoto Ups, didn't see your response before now. I've merged your pull request and it's now published as v3.4.0 on npm

**Comment by kkushimoto** [comment]
Thank you so much!

**Pull request merged**